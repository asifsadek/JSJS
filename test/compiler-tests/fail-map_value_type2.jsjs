val map = {
    1 : (/\(n) => n % 2 == 0),
    2 : (/\(n) => empty?(n))
};
