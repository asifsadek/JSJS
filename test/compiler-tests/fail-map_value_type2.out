Type error: expected value of type 'list A', got a value of type 'num' instead
